apps/storybook/src/NewsSectionDashboard.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import { NewsSection } from '@asap-hub/react-components';
+import { ComponentProps } from 'react';
+import { text } from '@storybook/addon-knobs';
+
+export default {
```

Comment:
Why is there a new storybook for an existing component? Like the component is called NewsSection so the convention is that it should be NewsSection.stories.tex

Reply:
The first reason was because other dashboard "parts" are on specific stories (PastEventsDashboard and UpcomingEventsDashboard) so I thought on following the same pattern

The second reason is because the news dashboard section is not only the NewsSection but also the View All link which can't be inside NewsSection because the latter is being used in other places like DiscoverNews. That might be a reason to create a new component only for the dashboard like DashboardNewsSection but I also thought would be additional effort for a small case

The third reason is I thought it wouldn't be needed a specific story for NewsSection as it's been used in DiscoverNews and there's no story for it also

Reply:
PastEventsDashboard and Upcoming Events Dashboard are two separate components.

Generally we put header and description in the relevant page layout as the card content themselves tend to be more reusable. I think you will find all the other components on the dashboard page do not have the Title and Description/Short Text within them/ 

The dashboard was created very quickly by the old project lead in order to meet contractual obligations so it's not the best example of how things should be done.